assignments/005-slurm-and-bwa-mem2/sbatchy.txt

```diff
@@ -0,0 +1,32 @@
+# Commands used to launch scripts as part of assignment 005-slurm-and-bwa-mem2
+
+# Trimming reads
+
+# First, set permissions to include execute permission
+[login11: con-gen-csu]--% chmod u+x assignments/005-slurm-and-bwa-mem2/trim_all.sh
+
+# Second, submit job to sbatch
+[login11: con-gen-csu]--% sbatch assignments/005-slurm-and-bwa-mem2/trim_all.sh
+
+
+# Create reference indexes to do mapping with bwa-mem2
+
+# First, set permissions to include execture permission
+[login11: con-gen-csu]--% chmod u+x assignments/005-slurm-and-bwa-mem2/index2.sh
```

Comment:
Hey Chris, One weird thing about sbatch is that the script your are running doesn't actually need to be executable.  (no problem if it is, but no problem if it is not---sbatch copies the whole script into a new file, essentially, and then makes that executable and runs that!